sdks/java/core/src/test/java/org/apache/beam/sdk/util/RowJsonTest.java

```diff
@@ -132,14 +132,10 @@ private static Object[] makeFlatRowTestCase() {
 
     private static Object[] makeLogicalTypeTestCase() {
       Schema schema =
-          Schema.builder()
-              .addLogicalTypeField(
-                  "f_passThroughString",
-                  new PassThroughLogicalType<String>(
-                      "SqlCharType", FieldType.STRING, "", FieldType.STRING) {})
-              .build();
+          Schema.builder().addLogicalTypeField("f_passThroughString", FixedString.of(10)).build();
```

Comment:
Why `FixedString.of(10)`? Does the padding behavior need to be tested here? (The common use case is in a SQL statement with string constants, which are allocated at the correct size and not padded.)

Reply:
The original test of "SqlCharType" is replaced. Could change it to VariableString.of(any) and it still tests l ogical yype test case and does not involve the padding behavior of FixedString logical type.